docs/source/development-guide/schema-deployment/schema-changes.rst

```diff
@@ -0,0 +1,17 @@
+Schema Change Process
+============
+
+1. All database changes will be approved via pull request by Sean Goggins, following these basic principles:
```

Comment:
Please add your GitHub username so that new contributors will know who to ask if they have schema changes.